packages/ui/react-ui-table/src/components/Table/Table.stories.tsx

```diff
@@ -116,9 +116,19 @@ const makeColumns = (onUpdate?: ValueUpdater<Item, any>): TableColumnDef<Item, a
 // Tests
 //
 
+const MinimalTable = (props: any) => {
+  return (
+    <Table.Root>
+      <Table.Viewport classNames='fixed inset-0 overflow-auto'>
```

Comment:
Is `overflow-auto` built-in to this component or no?

Reply:
Not currently, I wanted to get a sense of how we were containing it before making any styling decisions. Currently just looking at it as a way to manage the ref to the scroll context.